packages/main/src/requests/response-helpers.ts

```diff
@@ -54,6 +55,30 @@ export function addHelpers(
     }
     return "";
   };
+  (response as EnhancedGenerateContentResponse).functionCall = () => {
```

Comment:
The Vertex Python design for this convenience method is `functionCalls`, (internal link: https://docs.google.com/document/d/1Y2ncToFbzd1Ti1WJOXQHHAvFzCbNWzZYVleoDbcLrDc/edit?tab=t.0#heading=h.f0d2sodbkxt3) anticipating multiple function calls being returned in the future (see b/319098239).

But, if our philosophy is that these convenience methods (this method and text()) are expected to be "simplest case" methods and that any more complex cases should be handled by the user diving into `response.candidates` and handling it themselves, this makes sense.

Reply:
For now I guess let's keep this function as is. When there will be a change with multiple functionCalls we can rethink that thing